modules/postgres/resources/postgres-ssl.conf

```diff
@@ -0,0 +1,80 @@
+# -----------------------------
```

Comment:
I have one question regarding the default ssl conf: will it work with different versions of the Postgres image? If a user sets the postgres version to 13, will it work the same? What will happen with new versions of Postgres? Must we maintain this file once new versions appear? If so, I'd advocate for removing that and letting users to provide their configs.

For our tests, I do see it valuable, as we control the versions, but not sure if we should put a fixed config file into the postgres config for all users of the module. Wdyt?

Reply:
Reverted to use the previous approach of passing in.

Reply:
Thank you! I think that makes more sense 🙏